src/background/index.ts

```
@@ -671,6 +682,84 @@ function handleGetTabsPort(
+            type: MessageType.WORKFLOW_STATUS,
+            payload: success ? { status: 'success' } : { status: 'error', error: 'Save failed' },
+            id
+          })
+        }
+      )
+    } else {
+      // Fallback to chrome.storage.local for dev
+      try {
+        const key = BROWSEROS_PREFERENCE_KEYS.PROVIDERS
+        chrome.storage?.local?.set({ [key]: JSON.stringify(config) }, () => {
+          try { langChainProvider.clearCache() } catch (_) {}
```

ESLint

[error] Empty block statement.

(no-empty)